_includes/openseeds-header.html

```diff
@@ -0,0 +1,78 @@
+<nav class="navbar is-fixed-top" role="navigation" aria-label="main navigation">
+    <div class="container">
+      <div class="navbar-brand">
+        <a class="navbar-item" href="{% link openseeds/index.md %}">
+            <img src="/images/logo.png" alt="OLS">
+        </a>
+        <a
+          role="button"
+          class="navbar-burger burger"
+          aria-label="menu"
+          aria-expanded="false"
+          data-target="top-menu"
+        >
+          <span aria-hidden="true"></span> <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+        </a>
+      </div>
+
+      <div id="top-menu" class="navbar-menu">
+        <div class="navbar-end">
+          <div class="navbar-item has-dropdown is-hoverable">
+            <div class="navbar-dropdown">
+              <a class="navbar-item" href="{% link openseeds/index.md %}#applications"> Applications </a>
+              <a class="navbar-item" href="{% link about.md %}"> The program itself </a>
+              <a class="navbar-item" href="{% link about.md %}#projects"> Projects </a>
+              <a class="navbar-item navbar-previous-title" href="{% link about.md %}#the-roles"> Role description </a>
+              <a class="navbar-item" href="{% link about.md %}#mentees"> Mentees </a>
+              <a class="navbar-item" href="{% link about.md %}#mentors"> Mentors </a>
+              <a class="navbar-item" href="{% link about.md %}#experts"> Experts </a>
+              <a class="navbar-item" href="{% link about.md %}#facilitators"> Facilitators </a>
+              <a class="navbar-item navbar-previous-title" href="{% link openseeds/index.md %}#cohorts"> Cohorts </a>
+              <a class="navbar-item navbar-previous-title" href="{% link library.md %}"> Video Library </a>
```

Comment:
Maybe that could be a direct links in the navbar

Reply:
Done. It is more visible now in the navbar. Thanks a lot!